runtime/compiler/control/CompilationThread.cpp

```diff
@@ -1041,8 +1045,47 @@ TR::CompilationInfo::CompilationInfo(J9JITConfig *jitConfig) :
    _compReqSeqNo = 0;
    _newlyExtendedClasses = NULL;
    _chTableUpdateFlags = 0;
+#if defined(JITSERVER_SUPPORT)
+   _sslKeys = NULL;
+   _sslCerts = NULL;
+#endif
+   }
+
+#if defined(JITSERVER_SUPPORT)
+void TR::CompilationInfo::addJITServerSslKey(std::string key)
+   {
+   printf ("\n%s\n", __PRETTY_FUNCTION__);
```

Comment:
This printf must go away

Reply:
This is only here for debugging the crash I'm seeing. will be removed in the final version.